### PR TITLE
US-8906 Added cap generation for tvOS

### PR DESCRIPTION
**Pull request by simongranger**

### CODE REVIEW

File: lib/yi_appium_caps_util.rb [hunk 1]
```diff
@@ -213,6 +219,57 @@ def update_mac_caps (output_data)
       raise "An error of type #{ex.class} happened, message is #{ex.message}"
     end
 
+    def update_tvos_caps (output_data)
+      #Replace value of deviceName
+      output_data['caps']['deviceName'] = "Apple TV"
+      output_data['parameters']['formFactor'] = "10foot"
+
+      if (output_data['caps']['skipYouiEngineAppAddress'] != nil)
+        puts
```
Comment (lukeyoui) [inline lib/yi_appium_caps_util.rb]: Is this an extra puts?

File: lib/yi_appium_caps_util.rb [hunk 2]
```diff
@@ -167,6 +172,7 @@ def update_ios_caps (output_data)
           puts 'IP Address: ' + ip
           if (ip == "")
             ipcounter +=1
+            sleepSec +=5
           else
             #Found it!  
             ipcounter = 999
```
Comment (lukeyoui) [inline lib/yi_appium_caps_util.rb]: I know its not changed in your PR, but do you think a "return" would work just as well as setting the counter to 999? That way we wouldn't have to check the loop condition again.

File: lib/yi_appium_caps_util.rb [hunk 3]
```diff
@@ -213,6 +219,57 @@ def update_mac_caps (output_data)
...
+        sleepSec = 7
+        while ipcounter <= 5 do
+          puts "Try \# #{ipcounter}"
+          puts "Sleep: #{sleepSec} sec"
+          logs = %x[ios-deploy --justlaunch --bundle #{myApp} & (idevicesyslog) & sleep #{sleepSec} ; kill $!]
+          File.write('logs.txt', logs)
+          #Getting ip from file
+          ip = %x[grep -Eo "IPAddress.*" logs.txt | head -1 | awk '{print $3}'| tr -d '"']
+          #Remove whitespace
+          ip = ip.strip
```
Comment (bcturnerYOUI) [inline lib/yi_appium_caps_util.rb]: Could you append `.strip` to the end of line 249 to avoid this step?

File: lib/yi_appium_caps_util.rb (hunk 1 above)
Comment (simongranger) [reply lib/yi_appium_caps_util.rb]: It's meant to add an empty line

**Review by lukeyoui — APPROVED** [review]

File: lib/yi_appium_caps_util.rb (hunk 3 above)
Comment (simongranger) [reply lib/yi_appium_caps_util.rb]: didn't work

**Pull request merged**